demo/debug_camera.hpp

```diff
@@ -110,63 +110,63 @@ class DebugCamera : public wr::CameraNode
 		{
 			if (key == 0x57) // W
 			{
-				m_forward_axis += 1;
+				m_forward_axis += -1;
 			}
 			if (key == 0x53) // S
 			{
-				m_forward_axis += -1;
+				m_forward_axis += 1;
 			}
-			if (key == 0x44) // A
+			if (key == 0x41) // A
```

Comment:
We should probably create an enum for the keys sometime soon...

Reply:
This can be replaced by 'A' be sure to match case.

Reply:
Oh yeah, didn't think of that. 👍